examples/TiO2-RSS/README.md

```diff
@@ -3,14 +3,14 @@ we use asap tools to do some analysis and clustering tasks
 
 #
 * step 1
-gen_soap_descriptors.py -fxyz rss.xyz -fdict knownphases.xyz --prefix TiO2 --rcut 4 --n 4 --l 4 --g 0.5
+python ../../scripts/gen_soap_descriptors.py -fxyz rss.xyz -fdict knownphases.xyz --prefix TiO2 --rcut 4 --n 4 --l 4 --g 0.5
```

Comment:
We decided to just use the name of the script, instead of the relative path in the examples,
because we expect people to install it